.github/workflows/covector.yml

```diff
@@ -0,0 +1,156 @@
+name: Covector
+on:
+  push:
+    branches:
+      - dev
+    paths-ignore:
+      - 'specs/**'
+      - 'documentation/**'
+  pull_request:
```

Comment:
I might be wrong but does this need to run on PRs? The `status` job just seems to [provide outputs](https://github.com/jbolda/covector/tree/covector-v0/packages/action#outputs) for `version-or-publish` which should only run on push events

Reply:
Hmm.